src/components/layer-manager/layer-time/layer-time-slider.vue

```diff
@@ -71,32 +80,64 @@ function computeTimeValues() {
   return timeValueList
 }
 
-function onChange(event: Event) {
+function sliderValueToTimeValueMapper(selectedValue: number) {
+  const selectedTimeValue = timeValues.value[selectedValue]
+
+  return new Date(selectedTimeValue).toISOString()
+}
+
+function onChangeSingleDate(event: Event) {
   const cursorIndex = parseInt((event.target as HTMLInputElement).value)
-  const selectedTimeValue = timeValues.value[cursorIndex]
+  const newDate = sliderValueToTimeValueMapper(cursorIndex)
+
+  emit('changeTime', newDate, props.layer.currentTimeMaxValue)
```

Comment:
Is the max value as a second value useful ? wouldn't it be clearer if the max value in this event was "null" ?